python/perf-kernels/rmsnorm.py

```diff
@@ -110,6 +116,7 @@ def test_rmsnorm(M, N):
     rms_norm = torch.nn.RMSNorm(N, device='cuda')
     y_torch = rms_norm(x)
 
+    print(f"y_triton={y_triton}")
```

Comment:
Is it a debugging statement that must be removed before merging?

Reply:
Removed